SNIFFER/readme.md

```diff
@@ -0,0 +1,7 @@
+# this  script is created in python
```

Comment:
I guess you can come up with a better README. Have a look at READMEs of other contributed projects to get some idea.